src/backend/distributed/connection/remote_commands.c

```diff
@@ -353,9 +353,8 @@ LogRemoteCommand(MultiConnection *connection, const char *command)
 	}
 
 	ereport(NOTICE, (errmsg("issuing %s", ApplyLogRedaction(command)),
-					 errdetail("on server %s@%s:%d connectionId: %ld", connection->user,
-							   connection->hostname,
-							   connection->port, connection->connectionId)));
```

Comment:
We've been using connectionId heavily to debug/understand connection management issues.

Reply:
Good to know, that's a discussion for another time then. Dropping that commit from this PR